lib/ash/helpers.ex

```diff
@@ -597,4 +597,13 @@ defmodule Ash.Helpers do
   end
 
   def deep_merge_maps(_left, right), do: right
+
+  @doc false
+  def redact(value) do
```

Comment:
We should make this a compile time configuration, and use `Application.compile_env`. Then, conditionally define the function (for performance).

Reply:
I've changed it to use `Application.compile_env/3`.
However I don't prefer that because `Application.get_env/3` takes less than 1us and the guideline advises avoiding compile-time environments.

> In any case, compile-time environments should be avoided. Whenever possible, reading the application environment at runtime should be the first choice.

https://hexdocs.pm/elixir/Application.html#module-compile-time-environment

Reply:
Hmm...that's true, but this could get called thousands of times every time you inspect a list of records. My initial reason for using compile_env was to make it not possible to toggle on at runtime in production though.

Reply:
I agree :)